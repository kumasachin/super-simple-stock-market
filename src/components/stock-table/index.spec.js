import { render, screen, cleanup } from '@testing-library/react';
import Stocks from './';

describe("Stocks", () => {
  afterEach(cleanup);

  it('should take a snapshot', () => {
    const { asFragment } = render(<Stocks />);
    expect(asFragment(<Stocks />)).toMatchSnapshot()
  })
});