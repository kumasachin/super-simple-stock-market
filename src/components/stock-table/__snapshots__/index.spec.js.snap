// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stocks should take a snapshot 1`] = `
<DocumentFragment>
  <h2
    data-testid="stock-table-title"
  >
    Stocks
  </h2>
  <table
    class="tableFormat"
    data-testid="stock-table"
  >
    <thead
      data-testid="stock-thead"
    >
      <tr
        data-testid="stock-tr"
      >
        <th>
          Stock Symbol
        </th>
        <th>
          Type
        </th>
        <th>
          Last Dividend
        </th>
        <th>
          Fixed Dividend
        </th>
        <th>
          Par Value
        </th>
        <th>
          Price
        </th>
        <th
          width="15%"
        >
          Dividend Yield
        </th>
        <th
          width="15%"
        >
          P/E Ratio
        </th>
        <th
          width="15%"
        >
          Volume Weighted Stock Price
        </th>
        <th>
          Quantity
        </th>
        <th>
          Action
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</DocumentFragment>
`;
